sdk/php/src/Tests/Connection/CliDownloaderTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Connection;
+
+use DaggerIo\Connection\CliDownloader;
+use DaggerIo\Connection\ProcessSessionDaggerConnection;
+use PHPUnit\Framework\TestCase;
+
+class CliDownloaderTest extends TestCase
+{
+    public function testCliDownload()
+    {
+        $versionToDownload = '0.9.3';
```

Comment:
This version should only be set in a single place, so that we can bump it part of releasing: https://github.com/dagger/dagger/blob/bdaefc4d9769c949618b70e521aa4094e58d50b2/internal/mage/sdk/python.go#L196-L205